pkg/interface/src/logic/reducers/group-update.ts

```diff
@@ -103,7 +103,7 @@ export default class GroupReducer<S extends GroupState> {
       const resourcePath = resourceAsPath(resource);
       state.groups[resourcePath] = {
         members: new Set(),
-        tags: { role: {} },
+        tags: { role: { admin: new Set([window.ship]) } },
```

Comment:
My question is "does `addGroup` only apply to a group we create? What if we accept an invite to a group?"

Reply:
That comes in as an initialGroup update